<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginGame5GraphNodeInfoDialog</class>
 <widget class="QDialog" name="PluginGame5GraphNodeInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>256</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Игра 5</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="5" column="0" colspan="4">
    <widget class="QLabel" name="labelRelevantTile">
     <property name="text">
      <string>Фишка №:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QLabel" name="labelMoveDirection">
     <property name="text">
      <string>Перемещение:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="labelPathCostOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLabel" name="labelSolutionOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="labelRestPathCost">
     <property name="text">
      <string>Стоимость оставшегося пути:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="labelNodeNum">
     <property name="text">
      <string>Номер вершины:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="labelRestPathCostOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="labelRelevantTileOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="labelMoveDirectionOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="labelMoveCostOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="labelNodeNumOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="labelParentNodeNumOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="labelParentNodeNum">
     <property name="text">
      <string>Номер вершины родителя:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="labelPathCost">
     <property name="text">
      <string>Стоимость пути:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="labelMoveCost">
     <property name="text">
      <string>Стоимость применения правила:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QLabel" name="labelSolution">
     <property name="text">
      <string>Принадлежность к решению:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="buttonNext">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="buttonPrev">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5" rowspan="8">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="6">
    <widget class="Board" name="gameBoard" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Board</class>
   <extends>QWidget</extends>
   <header>app/rdo_studio/plugins/game5/src/board.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
